packages/atomic/src/components/atomic-result/atomic-result-cell-desktop.pcss

```diff
@@ -240,9 +240,4 @@
       display: none;
     }
   }
-
-  /* == Common styles == */
```

Comment:
No sure why that was there, @btaillon ?

Reply:
Strange, I have no idea. Perhaps an issue when merging something? This was supposed to be part of `atomic-result-with-sections.pcss`.

Reply:
i guess its should be empty if the class is there hmm,